frame/staking/src/pallet/impls.rs

```diff
@@ -534,10 +552,18 @@ impl<T: Config> Pallet<T> {
 						total = total.saturating_add(stake);
 					});
 
+				let others = WeakBoundedVec::<_, T::MaxNominatorRewardedPerValidator>::force_from(
+					others,
+					Some(
+						"Warning: The number of nominators is bigger than expected. \
+						A runtime configuration adjustment may be needed.",
+					),
```

Comment:
```suggestion
					Some("exposures.others"),
```

the idea here is to just give a scope, not the full message. The message generated by frame-support is descriptive enough.

Reply:
Done, and amended a couple of other places too.